packages/core/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -72,10 +74,7 @@ export const ErrorPage = ({
             Go back
           </Link>
           ... or if you think this is a bug, please file an{' '}
-          <Link
-            href="https://github.com/backstage/backstage/issues"
-            rel="noopener noreferrer"
-          >
+          <Link href={support.url} rel="noopener noreferrer">
```

Comment:
opening my eyes a bit, considering this (excellent!) add provides info on SUPPORT of the system as a whole, this link might be a link to anything that might provide more support. e.g., this could be like a support portal that then asks a team do you want to open an issue, setup your access, etc.

in that vein, maybe the wording here should be slightly tweaked a la "please contact support" or something a bit better than that which I can't think of right now, but does that make sense? point is, it might NOT end up exclusively being an "issue link" for all companies.

Reply:
@adamdmharvey - Agreed. I updated the wording. You can see the result in the screen capture in the updated PR description.